lib/xcprofiler/plugin.rb

```diff
@@ -57,8 +57,8 @@ def thresholds
     end
 
     def inline_mode
-      return false if @inline_mode.nil? == false
-      true
+      return true if @inline_mode.nil?
+      @inline_mode
```

Comment:
Returning value should be boolean. Could you cast the type?.

`!!@inline_mode`

Reply:
```
Offenses:
lib/xcprofiler/plugin.rb:61:7: C: Avoid the use of double negation (!!).
      !!@inline_mode
      ^
```
RuboCop is not happy with this. What is your suggestion?